gamestonk_terminal/common/newsapi_view.py

```diff
@@ -17,26 +17,42 @@ def news(term: str, num: int):
         term to search on the news articles
     num : int
         number of articles to display
+    s_from: str
+        date to start searching articles from formatted YYYY-MM-DD
     """
     # TODO: Add argument to specify news source being used
-    # TODO: Add argument to specify date start to search for articles
-    s_from = (datetime.now() - timedelta(days=7)).strftime("%Y-%m-%d")
+
+    if s_from:
+        show_newest = False
+
+    else:
+        show_newest = True
+        s_from = (datetime.now() - timedelta(days=7)).strftime("%Y-%m-%d")
```

Comment:
So I think the best way to do this would be to have these as default args.  basically add to the function:

```
def news(term: str, num: int, s_from: str=(datetime.now() - timedelta(days=7)).strftime("%Y-%m-%d"), show_newest:bool = True):
```
This way you can just have a show_newest flag in addition to the start date  and they aren't coupled together.

Reply:
Good point